### PR TITLE
Update Add-WsusDynamicCategory #368

### PR DESCRIPTION
**Pull request by ojrb**
I add in the notes, a reference to SCCM when the user need to solve a more complex sceanario

### CODE REVIEW

**Comment by ojrb** [comment]
This PR solve the issue #368

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/wsus/add-wsusdynamiccategory.md [hunk 1]
```diff
@@ -169,6 +169,10 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## NOTES
 
+This cmdlet is used to add Dynamic Categories to WSUS, based on the types needed (Computer Model, Device or Application).  The definition of Dynamic Categories in a WSUS implementation, helps to categorize the appliying of the updates to the diferent types availables.
```
Comment (JohanFreelancer9) [inline docset/windows/wsus/add-wsusdynamiccategory.md]:
```suggestion
This cmdlet is used to add Dynamic Categories to WSUS, based on the type of requirement (computer model, device or application).  The definition of Dynamic Categories in a WSUS implementation helps to categorize the application of updates to the different categories available. 
```
Please ensure that my edit does not compromise technical integrity.

File: docset/windows/wsus/add-wsusdynamiccategory.md [hunk 2]
```diff
@@ -169,6 +169,10 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## NOTES
 
+This cmdlet is used to add Dynamic Categories to WSUS, based on the types needed (Computer Model, Device or Application).  The definition of Dynamic Categories in a WSUS implementation, helps to categorize the appliying of the updates to the diferent types availables. 
+
+In some cases the user needs advanced automation when Dynamic Categories are used, for example if the user wants to download a specific device driver for a specific group of computers in the physical network, this problem needs more advanced automation on the use of Dynamic Categories and in this case the use of [System Center Configurations Manager](https://www.microsoft.com/en-us/cloud-platform/system-center-configuration-manager) is needed, configuring the [Software Update Point](https://docs.microsoft.com/en-us/sccm/sum/get-started/install-a-software-update-point) feature.
```
Comment (JohanFreelancer9) [inline docset/windows/wsus/add-wsusdynamiccategory.md]:
```suggestion
In some cases, you need advanced automation when using Dynamic Categories. If you want to download a specific device driver for a specific group of computers in the physical network, for example, advanced automation is required for the use of Dynamic Categories. In this case, the use of [System Center Configurations Manager](https://www.microsoft.com/en-us/cloud-platform/system-center-configuration-manager) is needed to configure the [Software Update Point](https://docs.microsoft.com/en-us/sccm/sum/get-started/install-a-software-update-point) feature.
```

**Comment by ojrb** [comment]
@e0i  Added the NOTES section eliminated, as requested for this PR

**Comment by ojrb** [comment]
@e0i I substituted the **### RELATED LINKS** by **## RELATED LINKS** as requested ! Please review and Merge !

**Pull request merged**